Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM python:2.7
-RUN apt-get update && apt-get install -y libmysqlclient-dev gettext libjpeg62-turbo-dev
+FROM debian:jessie
+
+EXPOSE 8000
+CMD ["./bin/run-prod.sh"]
+
+RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential python python-dev python-pip \
+                                               libmysqlclient-dev libxslt1.1 libxml2 libxml2-dev libxslt1-dev && \
+    rm -rf /var/lib/apt/lists/*
 
 WORKDIR /app
 
+# Pin a known to work with peep pip version.
+RUN pip install pip==6.0.0
```

Comment:
With the newest peep you should be able to get rid of this. It support up through pip 8. I figured we'd eventually move this to pip 8 with pipstrap, but don't need anything more in this PR.